pkg/util/common.go

```diff
@@ -1189,3 +1189,45 @@ func UpdateCustomResourceDefinition(apiExtensionClient *apiextensionsclient.Clie
 func DeleteCustomResourceDefinition(apiExtensionClient *apiextensionsclient.Clientset, name string) error {
 	return apiExtensionClient.ApiextensionsV1beta1().CustomResourceDefinitions().Delete(name, &metav1.DeleteOptions{})
 }
+
+// WaitUntilPodsAreReady will wait for the pods to be ready
+func WaitUntilPodsAreReady(clientset *kubernetes.Clientset, namespace string, labelSelector string, timeoutInSeconds int64) (bool, error) {
+	// timer starts the timer for timeoutInSeconds. If the task doesn't completed, return error
+	timeout := time.NewTimer(time.Duration(timeoutInSeconds) * time.Second)
+	// ticker starts and execute the task for every n intervals
+	ticker := time.NewTicker(10 * time.Second)
+	for {
+		select {
+		case <-timeout.C:
+			ticker.Stop()
+			return false, fmt.Errorf("the pods weren't ready - timing out after %d seconds", timeoutInSeconds)
```

Comment:
Make this more specific.  Like add the label selector and namespace in the error message.

Reply:
done